### ISSUE TITLE
big reputation status - alignment suggestion (preferences) 

### ISSUE THREAD

**Issue by Legendlul** [Lunaversitay/crident.store#2]
The feeling of looking at that reputation box isn't my best thing to be looking at cause of its size and most likely to be color too, I don't have any suggestions for any changes to it but at least resizing it or having a tab for it and not have the reputation below your rank status and replaced by something else.
![firefox_2018-12-26_14-31-03](https://user-images.githubusercontent.com/43793185/50444706-0d18b200-091b-11e9-98a7-7c851d7da039.png)

This info should be put at the end of the page.

![firefox_2018-12-26_14-33-25](https://user-images.githubusercontent.com/43793185/50444749-44875e80-091b-11e9-94ff-76a5c1659f25.png)

**Comment by Lunaversitay** [comment]
Yeah there's a couple pages that we missed with the height fix, I've gone ahead and fixed the height for that, will be out in the next update.
